cylc/flow/option_parsers.py

```diff
@@ -18,13 +18,26 @@
 import logging
 from optparse import OptionParser, OptionConflictError, Values, Option
 import os
+import re
+from ansimarkup import parse as cparse
 import sys
 from typing import Any, Dict, Optional, List
 
 from cylc.flow import LOG, RSYNC_LOG
 import cylc.flow.flags
 from cylc.flow.loggingutil import CylcLogFormatter
-from cylc.flow.terminal import format_shell_examples
+
+
+def format_shell_examples(string):
```

Comment:
@oliver-sanders, you may prefer to have this reverted to it's original home in `terminal.py`, in which case I'll put it back and put the imports for `workflow_files` at time of use, rather than at the top of the file.

Reply:
Prefer, yes, but it's really not important so can leave it like this.